bitmex-stream/src/lib.rs

```diff
@@ -42,6 +32,11 @@ pub fn subscribe_with_credentials<const N: usize>(
     subscribe_impl(topics, network, Some(credentials))
 }
 
+/// Connects to the BitMex websocket API, subscribes to the specified topics (comma-separated) and
+/// yields all messages.
```

Comment:
🙃 Ah, maybe I didn't explain it clearly. I think we do want a docstring on the public API (and a slightly different one per function)! 

It's just that the second part about the ping interval is pretty irrelevant for consumers, so it could just be demoted to a comment next to `subscribe_impl`.

Sorry for the confusion!

Reply:
ah, got it :)